remo/events/static/events/js/events_list.js

```diff
@@ -516,6 +516,9 @@ function send_query(newquery) {
     if (period === 'custom' || category || initiative) {
         EventsLib.adv_search_elm.show();
     }
+    else if (period === 'future' || period === 'past' || period === 'all') {
+        EventsLib.adv_search_elm.slideUp();
```

Comment:
I like that you used `slideUp` for hiding the filters. Could you adjust the line above and use `slideDown` instead of `show`. Just to have consistent UI.